client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-assembler-preview.scss

```diff
@@ -71,6 +72,24 @@
 	background-color: #f8f8f8;
 	opacity: 1;
 	z-index: -1;
+
+	h2 {
+		color: var(--studio-gray-100);
+		margin-bottom: 5px;
+		font-size: $font-title-small;
+		font-weight: 500;
+	}
+
+	span {
+		font-size: $font-body;
+		padding: 0 30px;
+		text-align: center;
+	}
+}
+
+.pattern-assembler-preview__icon {
+	margin-bottom: 40px;
```

Comment:
The icon without transparent background is 48px, so the margin-bottom should be `28px` (40px - (72px - 48px) / 2). Otherwise, it's too far away from the heading 😅

![image](https://user-images.githubusercontent.com/13596067/205806629-208e9e57-fb12-4b5a-909d-517d82884ef6.png)

Reply:
Done